src/main/java/org/jabref/cli/ArgumentProcessor.java

```diff
@@ -454,11 +454,17 @@ private boolean exportMatches(List<ParserResult> loaded) {
         // $ stands for a blank
         ParserResult pr = loaded.getLast();
         BibDatabaseContext databaseContext = pr.getDatabaseContext();
-        BibDatabase dataBase = pr.getDatabase();
 
         SearchPreferences searchPreferences = preferencesService.getSearchPreferences();
         SearchQuery query = new SearchQuery(searchTerm, searchPreferences.getSearchFlags());
-        List<BibEntry> matches = new DatabaseSearcher(query, dataBase).getMatches();
+
+        List<BibEntry> matches;
+        try {
+            matches = new DatabaseSearcher(query, databaseContext, preferencesService.getFilePreferences()).getMatches();
+        } catch (IOException e) {
+            System.err.println(Localization.lang("Error occurred when searching") + ": " + e.getLocalizedMessage());
```

Comment:
The other parts of the file use `LOGGER.error`, I think, it shold be used here, too.